src/components/base/materialcard/templates/templates.js

```diff
@@ -133,8 +135,66 @@ export function templateForVerticalWorkCard({ material }) {
   };
 }
 
-/**Used in Universe Page for Works */
+export function templateForUniverseInfoCard({ material }) {
+  const coverImageClassName = cx(
+    styles.cover,
+    styles.cover__universe_info_card
+  );
+
+  const href = getUniverseUrl(
+    material?.title,
+    material?.workId,
+    material?.universeNumber
+  );
+
+  return {
+    link_href: href,
+    ImageElement: material?.title
+      ? () => (
+          <div className={coverImageClassName}>
+            <Text tag="span" type="text2">
+              <Link border={{ bottom: { keepVisible: true } }} href={href}>
+                {Translate({
+                  context: "universe",
+                  label: "everything_in_universe",
+                  vars: [material?.title],
+                })}
+              </Link>
+            </Text>
+          </div>
+        )
+      : null,
+    // Styling
+    elementContainerClassName: cx(
+      styles.col_flex,
+      styles.col_flex__vertical_version
+    ),
+    relatedElementClassName: cx(
+      styles.related_element,
+      styles.related_element__vertical_version
+    ),
+    textClassName: cx(styles.text),
+    coverImageClassName: cx(styles.cover, styles.cover__vertical_version),
+  };
+}
+
+export function templateForUniverseSliderWork({ material }) {
+  const classNameOverride = {
+    elementContainerClassName: cx(
+      styles.col_flex,
+      styles.col_flex__vertical_version
+    ),
+  };
+
+  return templateForUniverseWorkBase({ material, classNameOverride });
+}
+
 export function templateForUniversePageWork({ material }) {
+  return templateForUniverseWorkBase({ material });
```

Comment:
this is not a way - det giver ingen mening med mindre du forudser at den ny template funktion skal gøre noget andet lige om lidt